okapi-core/src/main/java/org/folio/okapi/bean/RoutingEntry.java

```diff
@@ -185,6 +205,13 @@ public long getDelayMilliSeconds() {
     if (this.delay != null && unit != null) {
       long delayMilliSeconds = Integer.parseInt(this.delay);
       return delayMilliSeconds * factor;
+    } else if (schedule != null) {
+      Optional<Duration> duration = schedule.getNextDuration(ZonedDateTime.now());
```

Comment:
```suggestion
      Optional<Duration> duration = schedule.getNextDuration(ZonedDateTime.now(Clock.systemUTC()));
```
FOLIO is a multi-tenant system with tenants in different time zones. Therefore FOLIO always uses UTC in the back-end: https://dev.folio.org/guides/dates-and-times/

The crontab expression should be in UTC and should not depend on the locale timezone of the server where Okapi runs.

`0 2 * * *` (02:00 daily) and `2021-03-27T02:00+01:00[Europe/Paris]` yields `PT47H` because `2021-03-28` hasn't that hour due to the daylight savings switch. Using `2021-03-27T02:00Z` correctly yields `PT24H`.

Reply:
I'm fine with UTC by default. But I'm pretty sure this should be adjustable later OKAPI-969.